crowbar_framework/db/migrate/20150312081459_create_proposals.rb

```diff
@@ -0,0 +1,11 @@
+class CreateProposals < ActiveRecord::Migration
+  def change
+    create_table :proposals do |t|
+      t.string :barclamp, :null => false
+      t.string :name, :null => false
+      t.text :properties
```

Comment:
Btw, if we only keep the role assignment in the db, then maybe this needs to be a new table.

(Of course, all this requires quite some work in the other parts of the core to work, so this might be a later step in the branch)